azurerm_data_factory_trigger_schedule supports schedule and description (#13243)
hashicorp/terraform-provider-azurerm: website/docs/r/data_factory_trigger_schedule.html.markdown

* `pipeline_name` - (Required) The Data Factory Pipeline name that the trigger will act on.

* `description` - (Optional) The Schedule Trigger's description.

> * `schedule` - (Optional) The recurrence schedule for the trigger. A trigger with a specified frequency value alters its recurrence based on a recurrence schedule. The schedule property contains modifications for the recurrence that are based on minutes, hours, weekdays, month days, and week number. A `schedule` block as documented below.

Reviewer: ```suggestion
* `schedule` - (Optional) A `schedule` block as defined below, which further specifies the recurrence schedule for the trigger. A schedule is capable of limiting or increasing the number of trigger executions specified by the `frequency` and `interval` properties.
```

* `start_time` - (Optional) The time the Schedule Trigger will start. This defaults to the current time. The time will be represented in UTC.

* `end_time` - (Optional) The time the Schedule Trigger should end. The time will be represented in UTC.

* `interval` - (Optional) The interval for how often the trigger occurs. This defaults to 1.

* `frequency` - (Optional) The trigger frequency. Valid values include `Minute`, `Hour`, `Day`, `Week`, `Month`. Defaults to `Minute`.

* `pipeline_parameters` - (Optional) The pipeline parameters that the trigger will act upon.

* `annotations` - (Optional) List of tags that can be used for describing the Data Factory Schedule Trigger.

---

`schedule` supports the following:

> * `days_of_month` - (Optional) 	Day of the month on which the trigger runs. The value can be specified with a monthly frequency only.

Reviewer: ```suggestion
* `days_of_month` - (Optional) Day(s) of the month on which the trigger is scheduled. This value can be specified with a monthly frequency only.
```

* `days_of_week` - (Optional) Days of the week the trigger runs. The value can be specified only with a weekly frequency.

> * `hours` - (Optional) Hours of the day at which the trigger runs.

Reviewer: ```suggestion
* `hours` - (Optional) Hours of the day on which the trigger is scheduled.
```

* `minutes` - (Optional) Minutes of the hour at which the trigger runs.

* `monthly_occurrence` - (Optional) Days of the month on which the trigger runs. The value can be specified with a monthly frequency only. A `monthly_occurrence` block as documented below.

---

> `monthly_occurrence` supports the following:

Reviewer: ```suggestion
A `monthly_occurrence` block supports the following:
```

* `day` - (Required) The day of the week on which the trigger runs. For example, a `monthly_occurrence` property with a day value of `Sunday` means every Sunday of the month.

> * `occurrence` - (Optional) The occurrence of the specified day during the month. For example, a `monthly_occurrence` property with day and occurrence values of `Sunday, -1` means the last Sunday of the month.

Reviewer: ```suggestion
* `occurrence` - (Optional) The occurrence of the specified day during the month. For example, a `monthly_occurrence` property with `day` and `occurrence` values of `Sunday, -1` means the last Sunday of the month.
```
